modules/ROOT/pages/customsidebar.adoc

```diff
@@ -100,8 +103,11 @@ tinymce.PluginManager.add('myplugin', (editor) => {
     tooltip: 'My sidebar',
     icon: 'comment',
     onSetup: (api) => {
-      console.log('Render panel', api.element());
-    },
+     console.log('Render panel', api.element());
+      return () => {
+	   api.element().focus();
+	  };
+	},
```

Comment:
```suggestion
      console.log('Render panel', api.element());
      return () => api.element().focus();
    },
```
Just fixing indentation again. Likewise though focusing the element when it's destroyed doesn't make sense to me unless I'm missing something?

Reply:
As per above, I’ve committed @MitchCTiny’s suggested edits. They fix the indent character errors and change the sample to correct the focus error.